src/LandingScreen.js

```diff
@@ -8,23 +8,30 @@ export default class LandingScreen extends Component {
     super();
   }
 
+  displayTrailsByLocation = () => {
+    this.props.toggleLandingScreen();
+    this.props.getTrails();
+  }
+
   render() {
     return (
-      <div className="landing-screen">
-        
-        <h1>Trailhead</h1>
-        <select className="location-select">{this.props.parks.reduce((locationArr, location) => {
-          if (!locationArr.includes(location.usState)){
-            locationArr.push(location.usState)
-          }
-          return locationArr; 
-        }, []).map((usState) => {
-          return <option value={usState}>{usState}</option>
-        })
-      }
+      <div className="landing-screen-container">
+        <div className="landing-screen">
+          
+          <h1>Trailhead</h1>
+          <select className="location-select">{this.props.parks.reduce((locationArr, location) => {
+            if (!locationArr.includes(location.usState)){
+              locationArr.push(location.usState)
+            }
+            return locationArr; 
+          }, []).map((usState) => {
```

Comment:
next week, we should check this iterator pattern to see if there is a chance to refactor